# An array of points

geometry_msgs/PointStamped[] points
